### ISSUE TITLE
Write e2e and unit tests for 10-10d in vets-website

### ISSUE THREAD

**Issue by ndsprinkle** [department-of-veterans-affairs/va.gov-team#49134]
### Description

This can partially be done in parallel with the chapter tickets, although some e2e tests probably can't be written until the entire form is completed.

**Comment by tbaker1026** [comment]
Closing issue, will start end to end testing documentation with the first form.  Won't do it for 10-10d
